core/src/test/java/org/elasticsearch/index/query/GeoPolygonQueryBuilderTests.java

```diff
@@ -230,6 +230,22 @@ public void testParsingAndToQuery4() throws IOException {
         assertGeoPolygonQuery(query);
     }
 
+    public void testParsingAndToQuery5() throws IOException {
```

Comment:
It looks like these tests are a single polygon?  Can you also test that holes are parsed & executed correctly, and more than one polygon too?